client/src/django.js

```diff
@@ -37,6 +37,7 @@ const djangoClient = new Vue({
       await oauthClient.redirectToLogin();
     },
     async logout() {
+      await apiClient.post('/logout/', undefined, { withCredentials: true });
```

Comment:
I'm not sure how this will hold up long term, but this seems good enough for now. 👍